### PR TITLE
Lack resources in tools section for web development

### PR DESCRIPTION
**Pull request by Dev-Abhay01**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Resource URL: 
Fontjoy: https://fontjoy.com/
Typewolf: https://www.typewolf.com/

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [ *] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [ *] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by Dev-Abhay01** [comment]
please review and merge it... @jfmartinz

File: Web Development/README.md [hunk 1]
```diff
@@ -133,19 +133,32 @@ This category is a curated collection of valuable web development resources cont
...
+    <tr>
+        <td><a href="https://fonts.google.com/">Google Fonts</a></td>
+        <td>A vast collection of free, open-source fonts that you can easily integrate into your web projects.</td>
+    </tr>
+    <tr>
+        <td><a href="https://fontjoy.com/">Fontjoy</a></td>
+        <td>Generate font combinations with deep learning. Customize and fine-tune to find the perfect pairing.</td>
+    </tr>
+    <tr>
+        <td><a href="https://www.typewolf.com/">Typewolf</a></td>
```
Comment (jfmartinz) [inline Web Development/README.md:156]: Please removed this one. Someone wants to add this resource already in the section

File: Web Development/README.md [hunk 2]
```diff
@@ -133,19 +133,32 @@ This category is a curated collection of valuable web development resources cont
...
     <tr>
-      <td ><a href="https://www.fontpair.co/">Fontpair</a></td>
-      <td>Free fonts and font pairings to use for your next design project.</td>
+        <td><a href="https://www.fontpair.co/">Fontpair</a></td>
+        <td>Free fonts and font pairings to use for your next design project.</td>
+    </tr>
+    <tr>
+        <td><a href="https://fonts.google.com/">Google Fonts</a></td>
+        <td>A vast collection of free, open-source fonts that you can easily integrate into your web projects.</td>
+    </tr>
```
Comment (jfmartinz) [inline Web Development/README.md:150]: Removed this resource also

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Once you fixed these problemms, I will merged it. Thanks

**Comment by Dev-Abhay01** [comment]
Im just closing this pull request and raised it again please merge it then

**Pull request closed without merging**